checkov/common/output/csv.py

```diff
@@ -157,8 +159,23 @@ def persist_report_oss_packages(self, file_name: str, is_api_key: bool, output_p
             is_api_key=is_api_key,
         )
 
+    @staticmethod
+    def arrange_rows(rows: list[dict[str, Any]]) -> None:
```

Comment:
Have you tested the performance of this method on a real case, a big CSV file? I'm worried it would be pretty slow

Reply:
I tested it with CSV with 7000 rows, it finished in milliseconds.